docs/getting-started.md

```diff
@@ -0,0 +1,134 @@
+---
+sidebar_position: 1.1
+---
+
+# Getting Started
+
+The `@lukso/lsp6-signer.js` package is used to sign a transaction.
+
+The prefix `\x19LSP6 ExecuteRelayCall:\n` is added to prevent users from signing a transaction while thinking it's a message.
+
+- [GitHub Repository](https://github.com/lukso-network/tools-lsp6-signer)
+- [NPM Package](TODO)
+
+# Install
+
+```bash
+npm install @lukso/lsp6-signer.js
```

Comment:
I think this file is the same as the getting-started.md so above comments apply

Having duplicate file means we have to maintain the docs in two places so maybe we can just have one?